### PR TITLE
[ES] Add `osu!_pixel`

### PR DESCRIPTION
**Pull request by abbytec**
-Abby-

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/osu!_pixel/es.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - coordinate system
+---
+
+# osupixel
+
+Los **osupixels** son uno de los sistemas de coordenadas principales de osu! y aplican a todas las coordenadas de storyboards y [hit objects](/wiki/Hit_object). El tamaño de un osu!pixel cambia dependiendo de la resolucion seleccionada actualmente, pero es quivalente a un pixel cuando osu! se está ejecutando en resolución 640x480. Esta escala asegura una experiencia justa de juego para todos los jugadores.
+
+El escalado se alplica al sistema de coodenadas principal, que es utilizado para [osu! gameplay](/wiki/Gameplay). En constraste, las texturas son escaladas de manera que un [texture pixel](https://es.wikipedia.org/wiki/Texel_(gr%C3%A1ficos)) corresponde a un pixeel de pantalla a resolución 1024x768.
+
+<!-- TODO: needs links -->
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/osu!_pixel/es.md:14]:
```suggestion
```

**Comment by agatemosu** [comment]
To mark the self-check do it after sending the PR

File: wiki/Client/Beatmap_editor/osu!_pixel/es.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+tags:
+  - osu!pixel
+  - coordinate system
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/osu!_pixel/es.md:5]:
```suggestion
  - coordinate system
  - sistema de coordenadas
```

File: wiki/Client/Beatmap_editor/osu!_pixel/es.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# osupixel
+
+Los **osupixels** son uno de los sistemas de coordenadas principales de osu! y aplican a todas las coordenadas de storyboards y [hit objects](/wiki/Hit_object). El tamaño de un osu!pixel cambia dependiendo de la resolucion seleccionada actualmente, pero es quivalente a un pixel cuando osu! se está ejecutando en resolución 640x480. Esta escala asegura una experiencia justa de juego para todos los jugadores.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/osu!_pixel/es.md:10]:
```suggestion
Los **osupixels** son uno de los principales sistemas de coordenadas de osu! y aplican a todas las ubicaciones de los [objetos](/wiki/Hit_object) y las coordenadas de los storyboards. El tamaño de un osu!pixel cambia dependiendo de la resolucion seleccionada actualmente, pero es quivalente a un pixel cuando osu! se está ejecutando en resolución 640x480. Esta escala asegura una experiencia juego justa para todos los jugadores.
```

File: wiki/Client/Beatmap_editor/osu!_pixel/es.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+El escalado se alplica al sistema de coodenadas principal, que es utilizado para [osu! gameplay](/wiki/Gameplay). En constraste, las texturas son escaladas de manera que un [texture pixel](https://es.wikipedia.org/wiki/Texel_(gr%C3%A1ficos)) corresponde a un pixeel de pantalla a resolución 1024x768.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/osu!_pixel/es.md:12]:
```suggestion
El escalado se aplica al sistema de coordenadas principal que se utiliza para el [juego de osu!](/wiki/Gameplay). Por el contrario, las texturas se escalan de manera que un [téxel](https://es.wikipedia.org/wiki/Texel_(gráficos)) corresponde a un píxel de pantalla con una resolución de 1024x768.
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**